libdnf/hy-goal.h

```diff
@@ -52,6 +52,12 @@ typedef enum {
     DNF_IGNORE_WEAK              = 1 << 15
 } DnfGoalActions;
 
+typedef enum {
+    DNF_PACKAGE_TYPE_ALL              = 0,
+    DNF_PACKAGE_TYPE_AVAILABLE        = 1,
+    DNF_PACKAGE_TYPE_INSTALLED        = 2
+} DnfPackageKind;
```

Comment:
this can't be DnfPackageKind... this name stands more for `DNF_PACKAGE_RPM`, `DNF_PACKAGE_CONTAINER` and so on, so please rename it